packages/expo-notifications/ios/EXNotifications/Notifications/Emitter/EXNotificationsEmitter.m

```diff
@@ -33,9 +34,7 @@ @implementation EXNotificationsEmitter
 - (void)setModuleRegistry:(UMModuleRegistry *)moduleRegistry
 {
   _eventEmitter = [moduleRegistry getModuleImplementingProtocol:@protocol(UMEventEmitterService)];
-
   _notificationCenterDelegate = [moduleRegistry getSingletonModuleForName:@"NotificationCenterDelegate"];
-  [_notificationCenterDelegate addDelegate:self];
```

Comment:
Only reason I can think of needing this is if responses could somehow get swallowed between now and when we add a listener...but that is what `EXNotificationCenterDelegate`'s `_pendingNotificationResponses` is for! 

so yeah, I can't think of any reason we would need this either 👍